jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -92,6 +92,7 @@ def buildExclusions = [
 
   // AIX PPC64 ---------------------------------------------
   [ /aix71/,                          anyType,     lt(10)  ],
+  [ /aix72/,                          anyType,     lt(14)  ],
```

Comment:
We probably want to ensure we only build releases on either AIX 7.1 or 7.2 but not both?